etc/profile-m-z/qutebrowser.profile

```diff
@@ -26,8 +31,14 @@ whitelist ${DOWNLOADS}
 whitelist ${HOME}/.cache/qutebrowser
 whitelist ${HOME}/.config/qutebrowser
 whitelist ${HOME}/.local/share/qutebrowser
+whitelist /usr/share/qtbrowser
```

Comment:
Shouldn't this be qutebrowser?

```
whitelist /usr/share/qutebrowser
```

Reply:
yes. I don't have /usr/share/qtbrowser on my system. here's what the qutebrowser package installs in /usr/share:
```
~ ❯❯❯ pacman -Ql qutebrowser | grep /usr/share                                                                                                                                                                ✘ 130 
qutebrowser /usr/share/
qutebrowser /usr/share/applications/
qutebrowser /usr/share/applications/org.qutebrowser.qutebrowser.desktop
qutebrowser /usr/share/icons/
qutebrowser /usr/share/icons/hicolor/
qutebrowser /usr/share/icons/hicolor/128x128/
qutebrowser /usr/share/icons/hicolor/128x128/apps/
qutebrowser /usr/share/icons/hicolor/128x128/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/16x16/
qutebrowser /usr/share/icons/hicolor/16x16/apps/
qutebrowser /usr/share/icons/hicolor/16x16/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/24x24/
qutebrowser /usr/share/icons/hicolor/24x24/apps/
qutebrowser /usr/share/icons/hicolor/24x24/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/256x256/
qutebrowser /usr/share/icons/hicolor/256x256/apps/
qutebrowser /usr/share/icons/hicolor/256x256/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/32x32/
qutebrowser /usr/share/icons/hicolor/32x32/apps/
qutebrowser /usr/share/icons/hicolor/32x32/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/48x48/
qutebrowser /usr/share/icons/hicolor/48x48/apps/
qutebrowser /usr/share/icons/hicolor/48x48/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/512x512/
qutebrowser /usr/share/icons/hicolor/512x512/apps/
qutebrowser /usr/share/icons/hicolor/512x512/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/64x64/
qutebrowser /usr/share/icons/hicolor/64x64/apps/
qutebrowser /usr/share/icons/hicolor/64x64/apps/qutebrowser.png
qutebrowser /usr/share/icons/hicolor/scalable/
qutebrowser /usr/share/icons/hicolor/scalable/apps/
qutebrowser /usr/share/icons/hicolor/scalable/apps/qutebrowser.svg
qutebrowser /usr/share/man/
qutebrowser /usr/share/man/man1/
qutebrowser /usr/share/man/man1/qutebrowser.1.gz
qutebrowser /usr/share/metainfo/
qutebrowser /usr/share/metainfo/org.qutebrowser.qutebrowser.appdata.xml
qutebrowser /usr/share/qutebrowser/
qutebrowser /usr/share/qutebrowser/scripts/
qutebrowser /usr/share/qutebrowser/scripts/cycle-inputs.js
qutebrowser /usr/share/qutebrowser/scripts/dictcli.py
qutebrowser /usr/share/qutebrowser/scripts/hist_importer.py
qutebrowser /usr/share/qutebrowser/scripts/hostblock_blame.py
qutebrowser /usr/share/qutebrowser/scripts/importer.py
qutebrowser /usr/share/qutebrowser/scripts/keytester.py
qutebrowser /usr/share/qutebrowser/scripts/mkvenv.py
qutebrowser /usr/share/qutebrowser/scripts/open_url_in_instance.sh
qutebrowser /usr/share/qutebrowser/scripts/opengl_info.py
qutebrowser /usr/share/qutebrowser/scripts/utils.py
qutebrowser /usr/share/qutebrowser/userscripts/
qutebrowser /usr/share/qutebrowser/userscripts/README.md
qutebrowser /usr/share/qutebrowser/userscripts/add-nextcloud-bookmarks
qutebrowser /usr/share/qutebrowser/userscripts/add-nextcloud-cookbook
qutebrowser /usr/share/qutebrowser/userscripts/cast
qutebrowser /usr/share/qutebrowser/userscripts/dmenu_qutebrowser
qutebrowser /usr/share/qutebrowser/userscripts/format_json
qutebrowser /usr/share/qutebrowser/userscripts/getbib
qutebrowser /usr/share/qutebrowser/userscripts/kodi
qutebrowser /usr/share/qutebrowser/userscripts/open_download
qutebrowser /usr/share/qutebrowser/userscripts/openfeeds
qutebrowser /usr/share/qutebrowser/userscripts/password_fill
qutebrowser /usr/share/qutebrowser/userscripts/qr
qutebrowser /usr/share/qutebrowser/userscripts/qute-bitwarden
qutebrowser /usr/share/qutebrowser/userscripts/qute-keepass
qutebrowser /usr/share/qutebrowser/userscripts/qute-keepassxc
qutebrowser /usr/share/qutebrowser/userscripts/qute-lastpass
qutebrowser /usr/share/qutebrowser/userscripts/qute-pass
qutebrowser /usr/share/qutebrowser/userscripts/qutedmenu
qutebrowser /usr/share/qutebrowser/userscripts/readability
qutebrowser /usr/share/qutebrowser/userscripts/readability-js
qutebrowser /usr/share/qutebrowser/userscripts/ripbang
qutebrowser /usr/share/qutebrowser/userscripts/rss
qutebrowser /usr/share/qutebrowser/userscripts/taskadd
qutebrowser /usr/share/qutebrowser/userscripts/tor_identity
qutebrowser /usr/share/qutebrowser/userscripts/view_in_mpv
```

Reply:
@glitsj16

Reply:
@Dieterbe @kmk3 Correct! Looks like I messed that up. I double-checked and indeed, there isn't anything on the AUR either that uses that /usr/share/qtbrowser path.

I'll take care of it ASAP.

Side note: [Qt 6](https://www.qt.io/product/qt6) is being actively used now (at least on Arch Linux), so we should add /usr/share/qt6 to `whitelist-usr-share-common.inc`.